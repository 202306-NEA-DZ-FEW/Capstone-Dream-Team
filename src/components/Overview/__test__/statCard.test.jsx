import renderer from "react-test-renderer";

import StatCard from "../statCard";

it("renders correctly", () => {
    const tree = renderer.create(<StatCard />).toJSON();
    expect(tree).toMatchSnapshot();
});
