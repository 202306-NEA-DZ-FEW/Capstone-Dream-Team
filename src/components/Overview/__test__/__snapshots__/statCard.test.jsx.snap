// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col px-5 py-6 shadow-sm rounded-md border border-orange-100 bg-orange-50"
>
  <div
    className="flex"
  >
    <div
      className="flex items-center h-11 w-11 justify-center p-3 rounded-full bg-orange-400 bg-opacity-80"
    >
      <div
        className="text-2xl text-white"
      />
    </div>
    <div
      className="mx-2"
    >
      <h4
        className="text-2xl font-bold text-gray-700 p-1"
      />
    </div>
  </div>
  <div
    className="text-gray-500 overflow-hidden whitespace-nowrap text-base break-words pt-2"
  />
</div>
`;
